### PR TITLE
Create basic endpoint calls

### PR DESCRIPTION
**Pull request by jake-good**
Doesn't close issue just set up for end point calls

**Description**

Made templates for the api end point calls to both get and post bill payments. Converted the transaction list component to a class component to access life cycle methods.

**Testing**

Unable to test as just template setup that does not access actual back end api. Ensured front end still compiles and renders without error.
**Checklist**

- [x] I have tested my change
- [x] Code builds successfully and all tests pass
- [ ] Docs have been added/updated if needed

### CODE REVIEW

**Review by williamlixu — APPROVED** [review]
LGTM

File: split-webapp/split/src/pages/Split.js [hunk 1]
```diff
@@ -48,9 +47,24 @@ class Split extends Component {
...
+            total: cost,
             payments: paymentArray,
-            payed: ""
         }
+
+        this.createBill(bill);
+    }
+
+    createBill(data) {
+        return fetch('/api/bill_exec/create_bill', {
```
Comment (azizul-i) [inline split-webapp/split/src/pages/Split.js]: Is return required here?

Comment (pabloSushibar) [reply split-webapp/split/src/pages/Split.js]: Jake and I were not too sure on how to get data using the fetch method, so we took some inspiration from an example. Return in this case is not necessary, thank you for pointing it out!

File: split-webapp/split/src/components/TransactionList.js [hunk 2]
```diff
@@ -13,14 +13,34 @@ import "../App.css";
 function ListItemLink(props) {
   return <ListItem button component="a" {...props} />;
 }
- 
-export default function TransactionList() {
 
-const transaction = {
+class TransactionList extends React.Component {
```
Comment (vee-christella) [inline split-webapp/split/src/components/TransactionList.js:17]: Glad to see that we're using React Components now - is this applicable to all future components?

**Review by vee-christella — APPROVED** [review]

File: split-webapp/split/src/components/TransactionList.js (hunk 2 above)
Comment (pabloSushibar) [reply split-webapp/split/src/components/TransactionList.js:17]: Yeah we will be doing this from now on!

**Pull request merged**